keyboards/whtkbrd/config.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2022 Alexander Metz
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0001
+#define DEVICE_VER 0x0001
+
+#define MANUFACTURER whtk
+#define PRODUCT whtkbrd
+
+#define SPLIT_USB_DETECT
```

Comment:
This isn't needed.  ARM defaults to split usb detect. 
```suggestion
```

Reply:
Removed as suggested